packages/openactive-broker-microservice/src/core.js

```diff
@@ -886,7 +930,7 @@ function detectOpportunityType(opportunity) {
 }
 
 /**
- * @param {{[k: string]: any}} opportunity
+ * @param {import('./models/core').Opportunity} opportunity
```

Comment:
The thinking is that in future we could put more thorough typing on this object (especially given that it has been validated [Could we couple TS with validator I wonder 🤔]) and that would then immediately improve type-checking on all functions that act on opportunity data